pkr/cli/parser.py

```diff
@@ -103,7 +103,13 @@ def _create_kard(args):
     try:
         extra_features = args.features
         if extra_features is not None:
-            extras.update({'features': extra_features.split(',')})
+            extra_features = extra_features.split(',')
+            if 'features' in extras:
+                extra_features.extend(
+                    x for x in extras['features'] if x not in extra_features)
+                extras['features'] = extra_features
+            else:
+                extras.update({'features': extra_features})
```

Comment:
Add the ability to mix features in command line (-f) and features in meta file (-m)

Reply:
:+1:

Reply:
maybe add it through a set, to avoid having the same feature twice

Reply:
set is messing with the order, I prefer the deterministic way, but it is covered here.

Reply:
deterministic to be able to compare meta files actually.

Reply:
With new version, we should be able to handle order / unicity / merging. Will resolve the other conversation on the subject, most of the code is here.

Reply:
pkr kard update will need to be addressed, but I propose to handle it in a subsequent PR targeted to pkr kard update itself.